cimgen/languages/modernpython/templates/enum_class_template.mustache

```diff
@@ -0,0 +1,15 @@
+"""
+Generated from the CGMES 3 files via cimgen: https://github.com/sogno-platform/cimgen
```

Comment:
It's not only for CGMES 3!
See the other templates, e.g. cimpy_class_template.mustache:
"Generated from the CGMES files ..."